### ISSUE TITLE
Vulkan swapchain warning message printed when replaying DX12 capture

### ISSUE THREAD

**Issue by davidd-lunarg** [LunarG/gfxreconstruct#1402]
The warning message
```
[gfxrecon] WARNING - Rendering was restricted to surface index 0, but a surface was never created for that index; replay created 0 surface(s)
```
is logged when replaying any DX12 capture. This error comes from `VulkanSwapchain::Clean()` and does not appear to be a valid warning for DX12 replay.

**Comment by nickdriscoll-lunarg** [comment]
Unable to reproduce issue. Replayed a capture of the D3D12 sample mentioned in #1465 and a commercial title without encountering this warning.

**Comment by nickdriscoll-lunarg** [comment]
This issue was resolved in #1156 when VulkanSwapchain::options_surface_index_ was replaced by VulkanSwapchain::swapchain_options_.
